apps/src/sites/hourofcode.com/pages/public/index.js

```diff
@@ -1,20 +1,95 @@
-var google;
-var thanksUrl;
-var signupErrorMessage;
-var censusErrorMessage;
-var hocYear;
+/* globals google thanksUrl signupErrorMessage censusErrorMessage hocYear */
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import SchoolAutocompleteDropdown from '@cdo/apps/templates/SchoolAutocompleteDropdown.jsx';
+import SchoolNotFound from '@cdo/apps/templates/SchoolNotFound.jsx';
+
+var schoolData = {
+  nces: '',
+  schoolName: '',
+  schoolCity: '',
+  schoolState: '',
+  schoolZip: '',
+  schoolType: '',
+  showDropdownError: false,
+  showSchoolNotFoundError: false
+};
+
+function renderSchoolDropdown() {
+  ReactDOM.render (
+    <SchoolAutocompleteDropdown
+      setField={schoolDropdownOnChange}
+      value={schoolData.nces}
+      showErrorMsg={schoolData.showDropdownError}
+    />,
+    $('#school-selector')[0]
+  );
+}
+
+function renderSchoolNotFound() {
+  ReactDOM.render (
+    <SchoolNotFound
```

Comment:
Chatted  in person, but documenting that we want the Event Location field to appear rather than `SchoolNotFound` component if "My school isn't listed." is selected because Event Location does a google maps look up that we need for the Hour of Code map.

Reply:
FYI Drew has done the work to get the Event Location field working, but I also confirmed with both Marina and Tanya that either the Event Location or the 5 School Address fields are fine. The highest priority is getting this onto HOC ASAP before we send a big email on Wednesday- can we please review this and get it ready to merge today?

Reply:
I've updated this to use the event location rather than SchoolNotFound.